internal/workflow/processing.go

```diff
@@ -1094,6 +1103,36 @@ func (w *ProcessingWorkflow) preprocessing(ctx temporalsdk_workflow.Context, tin
 	}
 }
 
+// poststorage executes the configured poststorage child workflows. It uses
+// a disconnected context, abandon as parent close policy and only waits
+// until the workflows are started, ignoring their results.
+func (w *ProcessingWorkflow) poststorage(ctx temporalsdk_workflow.Context, aipUUID string) error {
```

Comment:
I think we will want to capture the results of the poststorage workflow for display in the Dashboard at some point, which may require rethinking the child workflow policy, but I think this is fine for now.

Reply:
We may do that at some point. I initially considered that and I was planning to add a "parentClose" and "failParentWorkflow" config vars for each configured workflow. Something for the future, it will require to change the policy, how we wait for each workflow, add an expected result type, etc.